pkg/fleet/installer/default_packages.go

```diff
@@ -30,6 +30,14 @@ var defaultPackagesList = []defaultPackage{
 	{name: "datadog-agent", released: false, releasedWithRemoteUpdates: true},
 }
 
+var languageToPackage = map[string]env.ApmLibLanguage{
+	"datadog-apm-library-java":   "java",
+	"datadog-apm-library-ruby":   "ruby",
+	"datadog-apm-library-js":     "js",
+	"datadog-apm-library-dotnet": "dotnet",
+	"datadog-apm-library-python": "python",
+}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 Suggestion:** ‏To retroactively support any languages we may add in the future (php, go...) I'd rather avoid hardcoding this, can we make it a function instead?